### PR TITLE
run linter in CI

### PR DESCRIPTION
**Pull request by zramsay**
and address a few fixes

### CODE REVIEW

File: cmd/utils/parse_test.go [hunk 1]
```diff
@@ -14,15 +14,16 @@ import (
...
+var defaultGenesis = func() *core.Genesis {
 	g := new(core.Genesis)
 	if err := json.Unmarshal(defaultGenesisBlob, g); err != nil {
 		log.Fatalf("parsing defaultGenesis: %v", err)
 	}
 	return g
 }()
 
-func bigString(s string) *big.Int {
+func bigString() *big.Int {
```
Comment (odeke-em) [inline cmd/utils/parse_test.go]: I am not sure we should change this, as it is there for convenience despite not using another string except "10000000000000000000000000000000000"

**Review by odeke-em — APPROVED** [review]
Thank you @zramsay, LGTM except for one request.

File: cmd/utils/parse_test.go (hunk 1 above)
Comment (zramsay) [reply cmd/utils/parse_test.go]: i just go by the linting suggestions. if you think it's better to `// nolint` then we can

Comment (adrianbrink) [reply cmd/utils/parse_test.go]: Let's keep it how it is and add `// nolint`.

**Review by adrianbrink — DISMISSED** [review]
Just the change with the `bigString` function. Otherwise LGTM

**Comment by zramsay** [comment]
done

**Comment by adrianbrink** [comment]
@zramsay The problem remains, that circle ci does not have enough memory to run the entire linter. Either we have to turn some of these off for now or we really need to upgrade circle.

**Comment by zramsay** [comment]
weird. I've had no problems running linter in other repos ...

**Comment by adrianbrink** [comment]
Ethermint is a really large codebase with its dependencies. It might be something else. @zramsay Can you investigate this and see if you can get CircleCi to pass because I think this PR is super helpful for code quality.

**Pull request merged**